src/server/scripts/EasternKingdoms/SunwellPlateau/boss_eredar_twins.cpp

```diff
@@ -119,7 +119,9 @@ struct boss_sacrolash : public BossAI
 
             scheduler.CancelGroup(GROUP_SPECIAL_ABILITY);
             ScheduleTimedEvent(20s, [&] {
-                Unit* target = SelectTarget(SelectTargetMethod::MaxThreat, 1, 100.0f);
+                std::list<Unit*> targets;
+                SelectTargetList(targets, 5, SelectTargetMethod::MaxThreat, 0, 100.0f, true, false);
+                Unit* target = Acore::Containers::SelectRandomContainerElement(targets);
```

Comment:
Can `targets` be empty here? If so, test if targets is empty before proceeding


```suggestion
                Unit* target;
                if (!targets.empty())
                    target = Acore::Containers::SelectRandomContainerElement(targets);
```

https://github.com/Tereneckla/azerothcore-wotlk/blob/twin-target/src/common/Utilities/Containers.h#L130
> container cannot be empty